civihr_default_theme/assets/sass/partials/components/_scrollbar.scss

```diff
@@ -0,0 +1,17 @@
+// Custom scrollbar
+// --------------------------------------------------
+.chr_custom-scrollbar {
+    &::-webkit-scrollbar {
+        height: 10px;
```

Comment:
_(Applies for every previous/next instance)_

Could you put this in a custom variable?